integreat_cms/cms/models/languages/language.py

```diff
@@ -108,6 +109,23 @@ class Language(AbstractBaseModel):
             _("This is used in exported PDFs."),
         ),
     )
+    social_media_webapp_title = models.CharField(
+        max_length=100,
+        blank=False,
+        default=settings.BRANDING_TITLE,
```

Comment:
```suggestion
        blank=True,
```
I think adding the dynamic branding title as default value here is not ideal since it would require a database migration each time we change the setting.